scripts/meetingbaas.py

```
@@ -9,38 +9,55 @@
+from pipecat.transports.network.websocket_client import (
     WebsocketClientParams,
     WebsocketClientTransport,
 )
-from pipecat.audio.base import BaseAudioResampler
 
 from config.persona_utils import PersonaManager
 from config.prompts import DEFAULT_SYSTEM_PROMPT
 from meetingbaas_pipecat.utils.logger import configure_logger
+import sys
+import logging
+from pipecat.observers.loggers.debug_log_observer import DebugLogObserver
```

Ruff (0.11.9)

`pipecat.observers.loggers.debug_log_observer.DebugLogObserver` imported but unused

Remove unused import: `pipecat.observers.loggers.debug_log_observer.DebugLogObserver`

(F401)